gammapy/scripts/image_pipe.py

```diff
@@ -3,7 +3,11 @@
                         unicode_literals)
 import logging
 import numpy as np
+from scipy import interpolate
```

Comment:
Imports of optional dependencies like scipy need to be put in the function or method where it's used.
Build fail: https://travis-ci.org/gammapy/gammapy/jobs/123775918#L1428